src/components/InputItem/InputItem.module.css

```diff
@@ -0,0 +1,2 @@
+.input {
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5887770)